app/src/main/java/ch/epfl/qedit/backend/location/AndroidLocService.java

```diff
@@ -0,0 +1,47 @@
+package ch.epfl.qedit.backend.location;
+
+import android.Manifest;
+import android.content.Context;
+import android.content.pm.PackageManager;
+import android.location.Criteria;
+import android.location.LocationListener;
+import android.location.LocationManager;
+import androidx.core.app.ActivityCompat;
+
+public class AndroidLocService implements LocationService {
+
```

Comment:
Why call it `AndroidLocService` and not `LocService` ?

Reply:
Because I have the Mock service, and the real service... I wanted to make the difference clear, but that might be some bad naming on my part. I have to think about another name ;-).